entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/update/EntityViewUpdaterImpl.java

```diff
@@ -480,7 +480,7 @@ public static ViewToEntityMapper createViewIdMapper(EntityViewManagerImpl evm, M
     }
 
     @SuppressWarnings("unchecked")
-    private static void buildComponentFlushers(EntityMetamodel metamodel, Class<?> rootType, String attributePrefix, String mappingPrefix, String accessorPrefix, Set<Attribute<?, ?>> attributes, Map<AttributeAccessor, BasicAttributeFlusher> componentFlushers) {
+    private static void buildComponentFlushers(EntityViewManagerImpl evm, EntityMetamodel metamodel, Class<?> rootType, String attributePrefix, String mappingPrefix, String accessorPrefix, Set<Attribute<?, ?>> attributes, Map<AttributeAccessor, BasicAttributeFlusher> componentFlushers) {
```

Comment:
Since you added the `EntityviewManagerImpl` to many methods now, could you please remove the `EntityMetamodel` parameter and instead access that in the methods via `evm.getMetamodel().getEntityMetamodel()`?

Reply:
Ok! Wasn't sure whether you'd rather remove `EntityMetamodel` or i.e. pass the `JpaProvider` rather than the `EVMI`.